### PR TITLE
blockdev_stream_cor:block stream on cor filter

### PR DESCRIPTION
**Pull request by aliang123**
Block stream on copy-on-read filter

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2017282

### CODE REVIEW

**Comment by ngu-niny** [comment]
(2/4) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.blockdev_stream_cor: PASS (179.08 s)
 (4/4) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_cor: PASS (189.08 s)

Acked-by: Nini Gu ngu@redhat.com

File: qemu/tests/cfg/blockdev_stream_cor.cfg [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    storage_pool = default
+    storage_type_default = directory
+
+    image_size_data1 = 2G
+    image_size_data1sn = ${image_size_data1}
+    image_format_data1 = qcow2
+    image_format_data1sn = qcow2
+    image_name_data1 = data1
+    image_name_data1sn = data1sn
+    image_copy_on_read_data1 = True
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_cor.cfg:26]: yes or on or true is better

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_cor.q35: PASS (81.32 s)

File: qemu/tests/cfg/blockdev_stream_cor.cfg (hunk 1 above)
Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_cor.cfg]: Have updated, thanks zhenchao.

**Comment by zhencliu** [comment]
LGTM

Let's wait for the vt patch

**Pull request merged**